platforms/ios/cmake/Toolchains/common-ios-toolchain.cmake

```diff
@@ -158,7 +188,7 @@ else()
 endif()
 
 # Include extra modules for the iOS platform files
-set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}/platforms/ios/cmake/Modules")
+#set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}/platforms/ios/cmake/Modules")
```

Comment:
It breaks iOS configuration.

Reply:
I was testing to see if I need to add something similar for visionOS but commenting out this line didn't break anything on the iOS build so I'm not sure its needed any more. I can add it back just in case

Reply:
OK - its back now